Moq.AutoMock.Tests/DescribeAsDisposable.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Linq;
+
+namespace Moq.AutoMock.Tests
+{
+    [TestClass]
+    public class DescribeAsDisposable
+    {
+        [TestMethod]
+        public void It_disposes_all_registered_disposable_instances()
+        {
+            TestDisposable test = new();
+            AutoMocker mocker = new();
+            mocker.Use(test);
+            IDisposable disposable = mocker.AsDisposable();
+
+            disposable.Dispose();
```

Comment:
Why the explicit call to dispose?
```suggestion
            using (mocker.AsDisposable())
            {
            }
```

Reply:
The idea was a clear separation for AAA. But I have no issue going with the explicit using.